dbt-snowflake/src/dbt/adapters/snowflake/catalogs/_native.py

```diff
@@ -0,0 +1,44 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from dbt.adapters.catalogs import (
+    CatalogIntegration,
+    CatalogIntegrationConfig,
+)
+from dbt.adapters.contracts.relation import RelationConfig
+
+from dbt.adapters.snowflake import constants, parse_model
+
+
+@dataclass
+class NativeCatalogRelation:
+    catalog_type: str = constants.DEFAULT_NATIVE_CATALOG.catalog_type
+    catalog_name: Optional[str] = constants.DEFAULT_NATIVE_CATALOG.name
+    table_format: Optional[str] = constants.NATIVE_TABLE_FORMAT
+    external_volume: Optional[str] = None
```

Comment:
Shouldn't these be omitted since they are no part of the DDL for Native? Seems to imply these could be something other than none (which as far as I know they can't)